src/components/v5/common/Table/partials/BurgerMenu.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import { BurgerMenuProps } from '../types';
+import Card from '~v5/shared/Card';
+import { useDetectClickOutside } from '~hooks';
+
+const displayName = 'v5.common.Table.partials.BurgerMenu';
+
+const BurgerMenu: FC<BurgerMenuProps> = ({
+  onToogle,
+  onToogleOff,
+  isMenuVisible,
+  onRemoveRow,
+  onDuplicateRow,
+}) => {
+  const { formatMessage } = useIntl();
+  const ref = useDetectClickOutside({
```

Comment:
There is registerContainerRef function in useToggle hook, so `useDetectClickOutside ` will be no longer needed